projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -855,6 +883,30 @@ function wpcom_launchpad_get_newsletter_subscriber_count() {
 	return (int) $total_subscribers;
 }
 
+/**
+ * Determines if Stripe has been connected.
+ *
+ * @return bool Whether or not Stripe account is connected.
+ */
+function wpcom_is_stripe_connected() {
+	require_lib( 'memberships' );
+	$blog_id  = get_current_blog_id();
+	$settings = (array) get_memberships_settings_for_site( $blog_id );
+	return boolval( $settings['connected_account_id'] );
+}
+
+/**
+ * Determines if paid membership plans exists
+ *
+ * @return bool Whether or not paid membership plans exist.
+ */
+function wpcom_has_paid_membership_plans() {
+	require_lib( 'memberships' );
+	$blog_id  = get_current_blog_id();
+	$settings = (array) get_memberships_settings_for_site( $blog_id );
+	return count( $settings['products'] ) > 0;
```

Comment:
Are we guaranteed that `$settings['products']` will be defined and an array? I feel we should be defensive against the data being undefined, especially as `count()` will throw exceptions for null values in newer versions of PHP.
```suggestion
	return isset( $settings['products'] ) && is_array( $settings['products'] ) && ( count( $settings['products'] ) > 0 ); // we could also use `array() !== $settings['products']` for the final check if we want
```

Reply:
Good call. Added the extra checks.